File: 3330. Find the Original Typed String I/find_the_original_typed_string_I.py
#T.C.->O(n)
#S.C.->O(1)
class Solution:
    def possibleStringCount(self, word: str) -> int:
        frequency = 1
        for i in range(1, len(word)):
            if word[i] == word[i - 1]:
                frequency += 1
        return frequency

File: 3330. Find the Original Typed String I/find_the_original_typed_string_I.cpp
//T.C.->O(n)
//S.C.->O(1)
class Solution {
public:
    int possibleStringCount(string word) {
         int frequency = 1; 

        for (int i = 1; i < word.size(); ++i) {
   
            frequency += (word[i] == word[i - 1]);
        }
      
        // Return the final frequency count
        return frequency;
    }
};
